packages/drivers/odsp-driver/src/createFile/createFile.ts

```diff
@@ -102,6 +104,20 @@ export async function createNewFluidFile(
 	fileEntry.docId = odspResolvedUrl.hashedDocumentId;
 	fileEntry.resolvedUrl = odspResolvedUrl;
 
+	odspResolvedUrl.context = resolvedUrl?.context;
+	odspResolvedUrl.appName = resolvedUrl?.appName;
+
+	if (shareLinkInfo?.createLink?.link) {
+		let newWebUrl = shareLinkInfo.createLink.link.webUrl;
+		newWebUrl = appendNavParam(
+			newWebUrl,
+			odspResolvedUrl,
+			odspResolvedUrl.dataStorePath ?? "/",
+			resolvedUrl?.codeHint?.containerPackageName,
```

Comment:
Use odspResolvedUrl here as well after you address the other comment to add containerPackageInfo to odspResolvedUrl.

Reply:
Added it.